src/gba/core.c

```diff
@@ -361,6 +364,36 @@ static void _GBACoreReloadConfigOption(struct mCore* core, const char* option, c
 		return;
 	}
 
+	if (strcmp("hwExtensions", option) == 0) {
+		if (mCoreConfigGetIntValue(config, "hwExtensions", &fakeBool)) {
+			core->opts.hwExtensions = fakeBool;
+			gba->hwExtensions.userEnabled = core->opts.hwExtensions;
+		}
+		return;
+	}
+	
+	if (strcmp("hwExtensionsFlags", option) == 0) {
+		const char hexDigits[] = "0123456789ABCDEF";
+		char hwExtensionsFlagsKey[] = "hwExtensionsFlags_XXXX";
```

Comment:
Why are you specifying this via aibitrary hex values anyway?

Reply:
As opposed to something like "hwExtensionMoreRam"? Because if I did that, everytime somebody wanted to create a new extension, they'd have to modify config.c  and core.c. 
While making the name generic and looping based on the number of extensions, they just need to add its id to the enum, and add the corresponding checkbox for the Qt ui (or config option for the rest of the platforms) to handle it.

Reply:
Editing core.c is sensible for this, and names are important since otherwise it's "idk something". config.c shouldn't be touched in this changeset anyway.

Reply:
Ok, then. So I'm guessing I should prefix this with "gba." too, right? Like "gba.hwExtensionMoreRam"?

Reply:
Something like that, though maybe more concise, like `gba.hwExt.extraRam` if you wanna namespace it.

Reply:
Fixed. I Ended up using "gba.extensions" for the global enable and "gba.ext.extraRam" for the extension, since I'm basically removing "hardware" from all names. Also changed "More RAM" to "Extra RAM" everywhere else.